File: Backend/TrackrAPI/TrackrAPI/Dtos/GestionExpediente/ExpedienteMuestrasGridDTO.cs
namespace TrackrAPI.Dtos.GestionExpediente
{
    public class ExpedienteMuestrasGridDTO
    {
        public int IdEntidadEstructuraTablaValor { get; set; }
        public int IdEntidadEstructura { get; set; }
        public DateTime? FechaMuestra { get; set; }
        public bool FueraDeRango { get; set; }
        public List<ExpedienteMuestrasRegistroDTO>? Registro { get; set; }



    }
}

File: Frontend/trackr-administrador/src/app/shared/dtos/gestion-expediente/expediente-muestras-grid-dto.ts
import { List } from "lodash";
import { ExpedienteMuestrasRegistroDTO } from "./expediente-muestras-registros-dto";

export class ExpedienteMuestrasGridDTO{
    idEntidadEstructura : number;
    idEntidadEstructuraTablaValor : number;
    fechaMuestra : Date;
    fueraDeRango : boolean;
    registro : ExpedienteMuestrasRegistroDTO[];

    constructor() {}
}